### DISCUSSION TITLE
Vue: Property setLocale does not exist on type

### DISCUSSION THREAD

**Discussion by artoryas** [nuxt-modules/i18n#2972] [Q&A]
I am using nuxt3, and for localization I am using useI18n() composable. I am calling setLocale method from it, it works, locale changes, but IDE giving me error that it does not exist in useI18n, even tho I have typings for other properties and method like "t, setLocalMessage, etc..."

```
const { setLocale } = useI18n();

setLocale('en');
```

Versions:
```
    "@nuxtjs/i18n": "^8.3.1",
    "nuxt": "^3.11.2",
```

<img width="745" alt="image" src="https://github.com/nuxt-modules/i18n/assets/39806297/69e830f9-f194-446b-b975-6e957af29ccd">

**Comment by danielgran** [comment]
same here, what the hell

**Comment by BobbieGoede** [comment]
Please open an issue with a reproduction project 🙏
